doc/manual/manual.tex

```diff
@@ -10501,6 +10501,91 @@ \subsubsection{Advection Stabilization Benchmarks}
+\begin{equation}
+\eta = \eta_r \left(\frac{d}{d_r}\right)^{m} \text{exp} \left[ \frac{E_a}{R}\left(\frac{1}{T}-\frac{1}{T_r}\right) \right] \text{exp} \left[ \frac{V_a}{R}\left(\frac{P}{T}-\frac{P_r}{T_r}\right) \right] A_{\eta}
+\label{eq:eta}
+\end{equation}
+where $d$ is the grain size, $m$ the grain size exponent (assumed to be 3), $R$ the gas constant, $E_a$ the activation energy and $V_a$ the activation volume. Subscripts $[X]_r$ refer to reference values, assumed to be $d_r = d = 1$~mm, $P_r = 1.5$~GPa and $T_r = 1200^{\circ}$C for the upper mantle. $A_{\eta}$ represents the extra reduction of viscosity due to an increase in $E_a$ near the solidus, expressed as
+\begin{equation}
+A_\eta(T^{\prime}) = 
+\begin{cases}
+1  & \text{for } T^{\prime} < T^{\prime}_{\eta} \\
+\text{exp} \left[-\frac{(T^{\prime}-T^{\prime}_{\eta})}{(T^{\prime}-T^{\prime}T^{\prime}_{\eta})} \text{ln}(\gamma)\right]& \text{for }T^{\prime}_{\eta} \leq T^{\prime} < 1 \\
+\gamma^{-1} \text{exp}(\lambda\phi) & \text{for }T^{\prime} \geq 1\\
+\end{cases}
+\end{equation}
+where $T^{\prime}_\eta$ is the homologous temperature above which activation energy becomes $E_a + \Delta E_a$, and $\gamma = 5$ is the factor of additional reduction. $\lambda\phi$ describes the direct effect of melt on viscosity, assumed to be negligible here. The solidus temperature, $T_s$, is fixed to a value of 1599 K at 50~km equivalent to a dry peridotite solidus \cite{Hirsch2000} and linearly increases below this depth according to
+\begin{equation}
+T_{s}(z) = 1599 + \frac {\partial T_s}{\partial z} (z - 50000)
+\label{eq:Ts}
+\end{equation}
+where $\frac {\partial T_s}{\partial z}$ is the solidus gradient. 
+
+In this benchmark, a 2D input ASCII file containing $V_S$ specified at 50 and 75 km depth is read in and converted to temperature using an initial temperature model that implements the $V_{S}(P,T)$ formulation detailed above. The parameters governing the relationship between $V_S$ and temperature are set to the values calibrated by Yamauchi \& Takei~\cite{YT16}, where $\mu_U^0 = 72.45$~GPa, $\frac{\partial{\mu_U}}{\partial{T}} = -0.01094$~GPa K$^{-1}$, $\frac{\partial{\mu_U}}{\partial{P}} = 1.987$,  $\eta_r = 6.22 \times 10^{21}$ Pa s, $E_a = 452.5$ kJ mol$^{-1}$, $V_a = 7.913 \times 10^{-6}$~m$^{3}$ mol$^{-1}$ and $\frac{\partial T_s}{\partial z} = 1.018$ K km$^{-1}$. As $V_S$ is a complex function of temperature, a Brent minimization algorithm is used to find optimal values. Fig.~\ref{fig:anelasticity} shows that the \aspect{} implementation of this parameterization can accurately recreate the results shown by Yamauchi \& Takei~\cite{YT16} in their Fig. 20.
```

Comment:
similar issues here with expressing units as mentioned above. (See also #3044 if you want to do it "right".)

Reply:
This should now be sorted.